src/Controller/Admin/PartyCrudController.php

```diff
@@ -66,6 +68,12 @@ public function configureFields(string $pageName): iterable
             ])
             ->addWebpackEncoreEntries('admin_party_identifier_lookup')
         ;
+        yield BooleanField::new('isUnderAddressProtection', 'Is under address protection')
+            ->hideOnIndex()
+            ->setFormTypeOptions([
+                'label' => '!'.$this->translator->trans('Is under address protection', [], 'case').'!',
```

Comment:
The `!`s should be a part of the translatable text.

In French there should be a space before `!` (`… !`) and in Spanish it would be `¡…!`. I know this is very theoretical, but I still have a point (I hope).

Reply:
Updated.